LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2023 Snowplow Analytics Ltd.
```

Comment:
We would still need to update this every year then? Why not make this `2013-present` as well?